static/js/tasks_browse.js

```diff
@@ -329,11 +329,14 @@ $(document).ready(function() {
         pybossaNotify('Your request is being executed. Please wait...', true, 'warning')
     });
 
+    var wizardHeight = $('#wizard-container').height() || 0;
```

Comment:
One idea to consider, since `wizardHeight` and `navbarHeight` are not being used elsewhere, could these two variables just be moved inside the `$('body').on('contextmenu')` block (i.e., move lines 332-333 to line 336.

Reply:
Ah.  I put them outside the callback since they should be constant per page load.  But I forgot that we're using es6 here, so I can `const` them.